### DISCUSSION TITLE
Where is the Firefly mobile (Android) wallet?

### DISCUSSION THREAD

**Discussion by serrq** [iotaledger/firefly#7787] [Feedback]
I looking for it everywhere in Github, but I didn't find its apk.

**Comment by begonaalvarezd** [comment]
Mobile was deprectated in July 2023, we dont maintain it any longer in the IOTA Foundation and it has been removed from the stores. If you want to use firefly you should use the desktop app, you can download it from the official website https://firefly.iota.org/

**Comment by serrq** [comment]
Sorry I don't use desktop. Mobile only.
